module-4/backend/week_two.md

```diff
@@ -4,8 +4,52 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-TBD  
+1. What is Webpack and why is it useful?
+
+Not sure exactly. Is it a compiler of some type for front end Javascript?
+
+2. When do you want to use event delegation?
+
+I don't know
+
+3. What's one difference between ES5 and ES6?
+
+ES5 uses prototypes when creating "classes", ES6 actually uses classes that is similar to other OOP languages.
+
+4. What's the deal with semi-colons in JavaScript?
+
+They're useful, but not really necessary any more. They're supposed to indicate the end of a command.
+
+5. How are you using the MVC design pattern in your Quantified Self project?
+
+MVC on the backend with the output being an API instead of a traditional view. On the front end, our model is supposed to be consuming the API instead of a database. I don't really know how a "controller" comes into play on the front end. FE is more confusing.
```

Comment:
you're implementing a MVC pattern with your backend specifically if you're creating models and controllers. the view piece is probably in the form of json